setup.py

```diff
@@ -307,8 +307,6 @@ def run(self):
     # Winfsptest requirements
     # We can't use `winfspy[test]` because of some pip limitations
     # - see pip issues #7096/#6239/#4391/#988
-    # Looking forward to the new pip dependency resolver!
-    'pywin32==227;platform_system=="Windows"',
```

Comment:
Shouldn't it be replaced with `winfspy[test]` as per the comment above?

Reply:
yup 3d92bf14327805db8c4ace600b4a44c7c394da3c